.github/workflows/test.yml

```diff
@@ -345,6 +358,14 @@ jobs:
         with:
           credentials_json: ${{ secrets.GOOGLE_CREDENTIALS }}
 
+      # Check if our destination compute instance exists and delete it
+      - name: Delete existing instance with same SHA
+        id: instance-exists
```

Comment:
Collision?

Reply:
Fixed with [3d349f7](https://github.com/ZcashFoundation/zebra/pull/3766/commits/3d349f73cef82166513b278b768bff05b17e5e87)

Reply:
If you mean because we have the same id, it should only collide within the same job

Reply:
Aha